Porting/release_managers_guide.pod

```diff
@@ -801,7 +801,7 @@ an entry has been added by F<corelist-perldelta.pl>.
 
 You should add pod links for GitHub issue references thusly:
 
- $ perl -p -i -e'BEGIN{undef $/}; s{(?:GH|github)\s*#(\d+)}{L<GH #$1|https://github.com/Perl/perl5/issues/$2>}img' pod/perldelta.pod
+ $ perl -gpi -e 's{\b(?:GH|github)\s*#(\d+)}{L<GH #$1|https://github.com/Perl/perl5/issues/$1>}ig' pod/perldelta.pod
```

Comment:
It seems to be complex enough to warrant a separate script for this task, maybe we should do that and reference the script in RMG to use that to cleanup perldelta's for new releases?

Reply:
Doesn't look that complex to me. In any case, your proposed change would be a separate PR, I feel.